administration/intro.adoc

```diff
@@ -103,40 +103,36 @@ Once Kubernetes is deployed, you will need to deploy the KubeVirt
 add-on. There are 2 ways to deploy the add-on to a cluster using the `kubectl` tool and
 manifest files:
 
-1. Stable flow
-+++++++++++++++
+1. Operator flow
+++++++++++++++++
 
-The stable flow installs KubeVirt using a manifest file, which contains all deployments and configurations of the
-KubeVirt core components:
+The recommended way to deploy KubeVirt is using the KubeVirt Operator and
+KubeVirt custom resource.
 
 [source,bash]
 ----
-$ RELEASE=v0.14.0
-$ kubectl apply -f https://github.com/kubevirt/kubevirt/releases/download/${RELEASE}/kubevirt.yaml
+$ export VERSION=v0.16.2
+$ kubectl apply -f https://github.com/kubevirt/kubevirt/releases/download/$VERSION/kubevirt-operator.yaml
+$ kubectl apply -f https://github.com/kubevirt/kubevirt/releases/download/$VERSION/kubevirt-cr.yaml
 ----
 
-2. Alternative flow (aka Operator flow)
-+++++++++++++++++++++++++++++++++++++++
-
-Since version 0.13.0 KubeVirt can be installed using the new KubeVirt Operator and a KubeVirt custom resource.
-
-_________________________________________________________________________
-Note: The Operator is considered to be experimental for the time being.
-There is currently no way to upgrade KubeVirt using this flow.
-_________________________________________________________________________
+Since version 0.14.0, you can use `kubectl` for waiting until all KubeVirt components are "Ready".
```

Comment:
If we're recommending 0.17.0 for the operator, then we can remove the "since 0.14.0" here.